src/pages/gen2/start/mobile-support/index.mdx

```diff
@@ -730,9 +733,31 @@ class _TodoScreenState extends State<TodoScreen> {
 
 This will create a random Todo every time a user clicks on the floating action button. You can see the `ModelMutations.create` method is used to create a new Todo.
 
-And update the `Text('TODO Application')` line in your **main.dart** file to use the `TodoScreen` widget.
+And update the `MyApp` widget in your **main.dart** file like the following:
+
+```dart
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+  @override
+  Widget build(BuildContext context) {
+    return Authenticator(
+      child: MaterialApp(
+        builder: Authenticator.builder(),
+        home: const Scaffold(
+          body: Column(
+            children: [
+              SignOutButton(),
+              Expanded(child: TodoScreen()),
```

Comment:
The overflow issue is gone, but on iOS the `SignOutButton` is hidden by the camera notch.

Reply:
Added SafeArea